geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -293,6 +251,17 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         if (evt.selected.length > 0) {
           var feature = evt.selected[0];
+
+          if (goog.isDefAndNotNull(feature.get('__source__')) &&
+              feature.get('__source__') == 'mymaps') {
```

Comment:
Can't you change this test to:

```js
if (feature.get('__source__') == 'mymaps') {
```

The test also won't pass if `feature.get('__source__')` is `null` or `undefined`.